websites/views.py

```diff
@@ -113,14 +112,16 @@ def get_queryset(self):
 
         if search is not None:
             # search query param is used in react-select typeahead, and should
-            # match on the title, name, and short_id
-            queryset = queryset.annotate(
-                search=SearchVector(
-                    "name",
-                    "title",
-                    "short_id",
+            # match on the title, name, or short_id
+            if " " in search:
+                # names and short_ids don't have spaces
+                queryset = queryset.filter(title__search=search)
```

Comment:
**Question:** If there are spaces in the query, do you think it would be preferable to OR-together the individual words? It might be surprising to people that searching for "Physics 8.02" yields no results.

I guess then we might get too many results, and maybe need to think about sorting them better, which might conflict with the existing ordering of this API. 🤷

Reply:
Good question, I'll ask Ferdi

Reply:
I took a different strategy, now "Music 21M.380", "Music 21m-380", "21M.380", and "21m-380" should all return the same results.